src/components/transport_manager/src/usb/libusb/usb_connection.cc.orig

```diff
@@ -0,0 +1,420 @@
+/*
```

Comment:
this file can be removed, correct?

Reply:
@iCollin Fixed in 4f9ac4004124a2b723a8c307b3b2023779d17e38. Please take a look.